backend/infrahub/git/tasks.py

```diff
@@ -108,3 +109,16 @@ async def git_branch_create(
     repo = await InfrahubRepository.init(id=repository_id, name=repository_name, client=client)
     async with lock.registry.get(name=repository_name, namespace="repository"):
         await repo.create_branch_in_git(branch_name=branch, branch_id=branch_id)
+
+
+@flow(name="artifact-definition-generate")
```

Comment:
Currently `message_bus.operations.requests.artifact_definition.generate` has the same flow name. Just to make sure, is it what we want?

Reply:
Yes that's fine
Would be good to come up with a naming convention for the flow & deployment but that's outside of the scope of this PR